src/main/java/com/ibm/crypto/plus/provider/OpenJCEPlusFIPS.java

```diff
@@ -67,12 +67,37 @@ public final class OpenJCEPlusFIPS extends OpenJCEPlusProvider {
     private static boolean ockInitialized = false;
     private static OCKContext ockContext;
 
+    private static final boolean isPlatformSupported;
+    private static final List<String> supportedPlatforms = List.of("Linux-amd64", "Linux-s390x",
+            "Linux-ppc64le", "Windows-amd64", "AIX-ppc64");
+
+    static {
+        String osName = System.getProperty("os.name");
+        String osArch = System.getProperty("os.arch");;
+        String currentPlatform = osName + '-' + osArch;
+        
+        if (supportedPlatforms.contains(currentPlatform)) {
```

Comment:
This could just be `isPlatformSupported = supportedPlatforms.contains(currentPlatform);`

Reply:
Done.